src/sentry/models/artifactbundle.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+
+
+@region_silo_only_model
+class ArtifactBundle(Model):
+    __include_in_export__ = False
+
+    bundle_id = models.CharField(max_length=32, null=True)
+    organization_id = BoundedBigIntegerField(db_index=True)
+    # TODO: define max length.
+    release_name = models.CharField(null=True)
```

Comment:
Couldn't this also be an integer field, like `dist_id`?

If we're going with strings, use the same maxlength as on `Release`: https://github.com/getsentry/sentry/blob/7c58e7cb0ec0b20052dabbeb243ac0e896985b96/src/sentry/models/release.py#L478

Reply:
Thanks! I was searching for the name of the release in the model and didn't think it could have been named `version` 😅.

Reply:
We cannot guarantee that a release exists yet. I think since we likely have one artifact per release, a string is fine. Alternatively one could deduplicate through another table but that seems pointless.

Reply:
I will add @beezz to check if the tables are fine.